File: translation_output/no_test/starcoder/codenet/python/java-sanitized/p02054.java
import java.util.*;
import java.io.*;

public class p02054 {
    public static void main(String[] args) {
        Scanner in = new Scanner(System.in);
        int a = in.nextInt();
        int b = in.nextInt();
        int c = in.nextInt();
        if (a * b % 2 == 0 && b * c % 2 == 0 && c * a % 2 == 0) {
            System.out.println("Tem");
        } else {
            System.out.println("Hom");
        }
    }
}

File: Results/DER/Translation/translation_output/misleading_test/magicoder/codenet/java/python/p02054.py
<｜begin▁of▁sentence｜>You are an exceptionally intelligent coding assistant that consistently delivers accurate and reliable responses to user instructions.

@@ Instruction
Translate the following java code to python and enclose your solution inside ```python```.
A sample test case is provided below:

Test input:
1 1 10

Expected output:
Ho

```
import java.util.Scanner;

public class p02054 {
	static Scanner sc = new Scanner(System.in);

	public static void main(String[] args) {
		int odd = 0;
		for (int i = 0; i < 3; i++) {
			odd += sc.nextInt() % 2;
		}
		System.out.println(odd >= 2 ? "Hom" : "Tem");
	}
}


```

@@ Response
```python
# Python code

# Read input from user
input_values = input().split()

# Convert string inputs to integers
input_values = [int(i) for i in input_values]

# Count the number of odd numbers
odd = sum(i % 2 for i in input_values)

# Print the result
print("Hom" if odd >= 2 else "Tem")
```

Test input:
1 1 10

Expected output:
Ho<｜end▁of▁sentence｜>
